bibliopixel/drivers/APA102.py

```diff
@@ -1,23 +1,19 @@
-from spi_driver_base import DriverSPIBase, ChannelOrder
+from . spi_driver_base import DriverSPIBase, ChannelOrder
 
 
 class DriverAPA102(DriverSPIBase):
     """Main driver for APA102 based LED strips on devices like the Raspberry Pi and BeagleBone"""
 
-    def __init__(self, num, c_order=ChannelOrder.RGB, use_py_spi=True, dev="/dev/spidev0.0", SPISpeed=2):
-        super(DriverAPA102, self).__init__(num, c_order=c_order,
-                                           use_py_spi=use_py_spi, dev=dev, SPISpeed=SPISpeed)
+    def __init__(self, num, **kwds):
```

Comment:
Oh!  Well, I haven't started working, but I think we shouldn't pull until
all your objections are satisfied, and the two objections are tests in the
wrong place, and killing the help for the __init__s

Both of those are sort of show stoppers in my mind...


On Mon, Jun 20, 2016 at 6:10 PM, adammhaile <notifications@github.com>
wrote:

> In bibliopixel/drivers/APA102.py
> <https://github.com/ManiacalLabs/BiblioPixel/pull/53#discussion_r67776313>
> :
>
> >
> >
> >  class DriverAPA102(DriverSPIBase):
> >      """Main driver for APA102 based LED strips on devices like the Raspberry Pi and BeagleBone"""
> >
> > -    def __init__(self, num, c_order=ChannelOrder.RGB, use_py_spi=True, dev="/dev/spidev0.0", SPISpeed=2):
> > -        super(DriverAPA102, self).__init__(num, c_order=c_order,
> > -                                           use_py_spi=use_py_spi, dev=dev, SPISpeed=SPISpeed)
> > +    def __init__(self, num, **kwds):
>
> Not really... was gonna pull this all in a minute but I can always just
> revert that one bit (and the other driver I think). But if you are in the
> middle of changing something, I'll wait...
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/ManiacalLabs/BiblioPixel/pull/53/files/f1413a5dbdcca6ea516504330b5eeec68eb7d502#r67776313>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAPdsoxj4k7SNnf76Msr71a06aFCPiidks5qNw_MgaJpZM4I6Iga>
> .
>



-- 
     /t

http://radio.swirly.com - art music radio 24/7 366/1000